packages/react/src/ActionList/ActionList.module.css

```
@@ -340,6 +340,21 @@
         }
       }
     }
+
+    &:where([data-active]),
+    &:where([data-is-active-descendant]) {
+      background: transparent;
+
+      /* provides a visual indication of the current item for Windows high-contrast mode */
+      outline: 2px solid var(--control-transparent-bgColor-selected);
```

```suggestion
      background: var(--control-transparent-bgColor-disabled);
```

I wonder if this might look a little cleaner? Just guessing, I haven't tried it